app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/singledisposals/asset_type_for_non_uk_residents.scala.html

```diff
@@ -118,7 +119,7 @@
             customErrorKey = Some(s"$key$userKey")
         )
 
-        @submitButton(messages(if (displayReturnToSummaryLink) "button.saveAndContinue" else "button.continue"))
+        @submitButton(messages(if (isAmmend) "button.continue" else if (displayReturnToSummaryLink) "button.saveAndContinue" else "button.continue"))
```

Comment:
this logic should be applied to all the triage question pages and to the representee pages. Looks like the saveAndContinue vs. conitnue logic was missing in some triage pages and was missing from the representee answers before but it should be there